sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_ManageModels.md

```diff
@@ -70,14 +70,14 @@ Console.WriteLine($"  Created on: {newCreatedModel.CreatedOn}");
 await client.DeleteModelAsync(newCreatedModel.ModelId);
 ```
 
-## Manage Custom Models Synchronously
+## Manage Models Synchronously
 
 Note that we are still making an asynchronous call to `WaitForCompletionAsync` for building a model, since this method does not have a synchronous counterpart.
 
 ```C# Snippet:FormRecognizerSampleManageModels
 var client = new DocumentModelAdministrationClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
 
-// Check number of custom models in the FormRecognizer account, and the maximum number of models that can be stored.
+// Check number of models in the FormRecognizer account, and the maximum number of models that can be stored.
```

Comment:
From what I've seen, the account properties don't include the prebuilt models in the count or limit, so I would prefer to leave the `custom models` in this sentence